templates/upstart/default/init.conf

```diff
@@ -44,7 +44,6 @@ limit stack {{{limit_stack_size}}} {{{limit_stack_size}}}
 {{/limit_stack_size}}
 setuid {{{user}}}
 setgid {{{group}}}
-console log # log stdout/stderr to /var/log/upstart/
```

Comment:
Personally, I kind of like upstart's `console log` capturing stuff to /var/log/upstart/whatever.log, and I think that mechanism is more amenable to external log rotation (like `logrotate`). The new mechanism in this PR only allows rotation by `copytruncate` which can cause data loss of the latest entries due to a race condition between the copy and the logging application.

Thoughts?